examples/ruby/request_data.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+def handler(context)
+  request = context.request
+  msg = <<~MSG
+    ---ENV---
+    #{request.env.map { |h| h.join('=') }.join("\n") }
+
+    ---HEADERS---
+    #{request.headers.map { |h| h.join(': ') }.join("\n") }
+
+    ---PARAMS---
+    #{request.params.map { |h| h.join('=') }.join("\n") }
+
+    --BODY--
+    #{request.body.read}
+  MSG
+
+  Rack::Response.new([msg]).finish
```

Comment:
yes... the thought was that by explicitly building the Rack::Response here, the example would illustrate that... some comments would probably help here...